_docs/setup/kubernetes/multicluster-install.md

```diff
@@ -0,0 +1,172 @@
+cluster.  For remotes, please set `pilotCfgStore: False`.
+
+**Important**: The implementation only uses the
+`config.istio.io/pilotCfgStore` and `config.istio.io/accessConfigFile`
+annotations, although every other annotation and spec is validated for
+correct syntax.  They may be set to dummy values, as long as they are
+syntactically correct.
+
+If the prerequisites are met, the credentials for each Kubernetes cluster
+will also be present in `$HOME/multicluster`.
+
+Assemble and create the configmap from these files in the Kubernetes cluster
+where the Istio control plane is operational:
+
+```bash
+kubectl create configmap -n istio-system multicluster -f $HOME/multicluster
+```
+
+## Deploy on the remote cluster
+
+Multicluster makes **one** Istio service mesh from all Kubernetes clusters
+that participate in multicluster.  Multicluster uses **one** Kubernetes
+cluster to run the Istio control plane.  **Any reasonable** number of remote
+Kubernetes clusters may be attached to the **one** Istio control plane.
+
+### Use kubectl with Helm to connect the remote cluster
+
+1. Use the helm template command on a remote to specify the Istio control plane:
+
+   ```bash
+   helm template install/kubernetes/helm/istio-remote --name istio-remote --set pilotEndpoint=`pod_ip_of_pilot_master` --set mixerEndpoint=`pod_ip_of_mixer_master` > $HOME/istio-remote.yaml
+    ```
+
+1. Instantiate the remote cluster's connection to the Istio control plane:
+
+   ```bash
+   kubectl create -f $HOME/istio-remote.yaml
+   ```
+
+### Alternatively use Helm and Tiller to connect the remote cluster
+
+1. If a service account has not already been installed for Helm, please
+install one:
+
+   ```bash
+   kubectl create -f install/kubernetes/helm/helm-service-account.yaml
+   ```
+
+1. Initialize Helm:
+
+   ```bash
+   helm init --service-account tiller
+   ```
+
+1. Install the Helm chart:
+
+   ```bash
+   helm install install/kubernetes/helm/istio-remote --name istio-remote --set pilotEndpoint=pod_ip_of_pilot_master --set mixerEndpoint=pod_ip_of_mixer_master`
+   ```
+
+### Mandatory Helm configuration parameters
+
+The `isito-remote` Helm chart requires the configuration of two specific variables defined in the following table:
+
+**Note** The `pilotEndpoint` and `mixerEndpoint` need to be resolvable via
+Kubernetes.
+
+| Helm Variable | Accepted Values | Default | Purpose of Value |
+| --- | --- | --- | --- |
+| `global.pilotEndpoint` | A valid IPv4 address | none | Specifies the Istio control plane's pilot Pod IP address |
+| `global.mixerEndpoint` | A valid IPv4 address | none | Specifies the Istio control plane's mixer Pod IP address |
+
+## Uninstalling
+
+** Note the uninstall method must match the installation method (`kubectl` or `tiller` based) **
+
+### Using kubectl to uninstall istio-remote
+
+* Uninstall an Istio remote:
+
+  ```bash
+  kubectl delete -f $HOME/istio-remote.yaml
+  ```
+
+### Using tiller to uninstall the istio-remote
+
+  ```bash
```

Comment:
Remove leaving spaces on this code block.

Reply:
nice catch - done.